android/app/src/main/java/com/allaboutolaf/MainActivity.java

```diff
@@ -23,6 +24,8 @@ public void onCreate(Bundle savedInstanceState) {
         if (!BuildConfig.DEBUG) {
             BugsnagReactNative.start(this);
         }
+        long size = 50L * 1024L * 1024L; // 50 MB
+        com.facebook.react.modules.storage.ReactDatabaseSupplier.getInstance(getApplicationContext()).setMaximumSize(size);
```

Comment:
I'm not entirely familiar with Java, but can't you just abbreviate this to ReactDatabaseSupplier since you imported the entire path?